CosmosApiMessageStatusChangeFeed/index.ts

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable @typescript-eslint/naming-convention */ // disabled in order to use the naming convention used to flatten nested object to root ('_' char used as nested object separator)
+import * as winston from "winston";
+import { Context } from "@azure/functions";
+import { AzureContextTransport } from "@pagopa/io-functions-commons/dist/src/utils/logging";
+import { TableClient, AzureNamedKeyCredential } from "@azure/data-tables";
+import * as KP from "../utils/kafka/KafkaProducerCompact";
+import { ValidableKafkaProducerConfig } from "../utils/kafka/KafkaTypes";
+import { getConfigOrThrow } from "../utils/config";
+import { IBulkOperationResult } from "../utils/bulkOperationResult";
+import { messageStatusAvroFormatter } from "../utils/formatter/messageStatusAvroFormatter";
+import { handleMessageStatusChange } from "./handler";
+
+// eslint-disable-next-line functional/no-let
+let logger: Context["log"] | undefined;
+const contextTransport = new AzureContextTransport(() => logger, {
+  level: "debug"
+});
+winston.add(contextTransport);
+
+const config = getConfigOrThrow();
+
+const messageStatusConfig = {

```

Comment:
Nice configuration way. Maybe we can put it in a utils function "WithTopic" to reuse it in future handler

Reply:
Done in [bfa0b33](https://github.com/pagopa/io-functions-elt/pull/35/commits/bfa0b33e2cb5a76359b69eefd9c25c4db5259aa6)